### DISCUSSION TITLE
Allow for multiple input files (i.e. a folder or wildcard path)

### DISCUSSION THREAD

**Discussion by DanielJackson-Oslo** [tailwindlabs/tailwindcss#7011] [Ideas]
Would love to be able to pass a folder or some wildcard path (/stylesheets/**.*.css) and then have Tailwind parse and watch all of those files. 

Usage:
 ```zsh
npx tailwindcss -i "./stylesheets/**.*.css" -o "build.css" --watch
```

Currently we are solving this by using a separate preprocessor to combine several css files into one before handing over to tailwind, but watching for changes both in the raw files that will be compiled into the tailwind input file and in the compiled tailwind input file feels unnecessary when tailwind already has an excellent `--watch` function!

I can take a stab at this if the team is positive :)

**Comment by matthiasg** [comment]
Considering tailwindcss guidelines explicitely mentions how tailwindcli is the simplest and fastest way and advises to use it ( https://tailwindcss.com/docs/installation) I also stumbled over this. I had initially though the `content` part in the config could just be extended. 

I do understand that doing that could cause ordering issues,  so a specific way to declare ordered input CSS files would definitely be helpful and expected in scenarios without postcss.

At the very least the documentation should mention that handling multiple css files is out of scope (which would also be fine) and refer to post css for this use-case.

**Comment by minervasm** [comment]
Why is this not supported already!?
